app/events/user_related_event.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class UserRelatedEvent < BaseEventStoreEvent
```

Comment:
@Martouta I assume this is to prepare for other user related events that might be added in the future? Not premature?

Reply:
Looking at `user_related_event_test.rb`I guess it's just the way we structure events

Reply:
Yes, it is the way that e structure events, and also the way that we structure Subscribers to those events (grouping by 'related' events 😄 )

Reply:
user related event makes sense.